issue_processing/jira_issue_processing.py

```diff
@@ -225,6 +259,69 @@ def merge_user_with_user_from_csv(user, persons):
     return issues
 
 
+def load_issue_via_api(issues, persons):
+    """For each issue in the list the history is added via the api
+
+        :param issues: list of issues
+        :param persons: list of persons from JIRA (incl. e-mail addresses)
+    """""
+
+    log.info("Load issue information via api...")
+    jira_project = JIRA(project_url)
+
+    for issue in issues:
+
+        min_date = issue['creationDate']
+        api_issue = jira_project.issue(issue['externalId'], expand='changelog')
+        changelog = api_issue.changelog
+        histories = list()
+
+        for change in changelog.histories:
```

Comment:
Please put a comment here about the time order in which the history entries are visited here: newest to oldest, oldest to newest, random? Without this information, it is hard to understand the logic in the following code parts.

Reply:
history entries are checked in time order from oldest to newest. I'll put a comment about this in the code